e2e-tests/cypress/e2e/experian/certificate-provider.cy.ts

```diff
@@ -0,0 +1,97 @@
+describe("Identify a Certificate Provider with a fraud checck", () => {
```

Comment:
End to end tests are a very inefficient way of testing logic: these three CP tests all run through several of the same steps, and are ultimately just testing the outcome of `FormProcessorHelper->processTemplate()` which already has unit tests checking the same thing. The donor tests are similarly doing three more scenarios which test the same thing again.

There's benefit in having an end-to-end scenario that checks the fraud check integration is working, but that already happens in the general journey flows (e.g. "lets you identify with National Insurance number").

So I reckon both of these cypress files can be deleted since they're not testing anything that isn't already covered in other tests.

Reply:
I'm not sure I wholly agree: the experian specific tests use the different LPAs that the new imposter mocks contain so that all the fraud check and KBV responses can be tested. I am happy though to move these tests out of the application so that they don't run in the pipeline, I'll just use them for local dev only. I do find them very useful for catching the occasional unexpected side effect.

Reply:
Ultimately though, I think it's still testing the same thing as the unit test: if the remote responds with a "REFER", it should do X.

Moving it out makes sense to me. If they're regularly catching things that wouldn't be noticed otherwise, that suggests we're missing unit tests that should have identified it.